### ISSUE TITLE
28F230J3D75 - Pin definition

### ISSUE THREAD

**Issue by thalesmaoa** [james-tate/arduino_rom_reader#1]
I really appreciate that you shared your code.
I'm playing with this [28F230J3D75](https://media.digikey.com/pdf/Data%20Sheets/Micron%20Technology%20Inc%20PDFs/MT28F(128,640,320)J3.pdf) memory and just want to dump it but I'm having trouble to port it.

I'm also willing to try an arduino due since it is 3.3V logic level and faster.

My chip has C[0..2] which I'm having trouble to identify which is RE, CE, CLE, ALE.
Is there a easy way to check it without try and error?

**Comment by james-tate** [comment]
Are you asking for pin definitions in my code? I have an #ifdef for the ATmega2560 for the pins I used on line 41 of the code. You should be able to drop in an Arduino Due port very easily right after this logic. 

Page 19 of your datasheet is the read operation for pulling data from the chip. I am not sure there is a quick way to tell which CE pin you will need to pull low. My approach would be to just try to read the chip's ID. If you perform a identifier code read on address 0x00 you should get 0x892C back. This should be an indication everything is connected correctly.

**Comment by thalesmaoa** [comment]
> Are you asking for pin definitions in my code?

Not really. Your code is very easy and intuitive, however, my datasheet is not.
For instance, here are your pin map:
![orig](https://user-images.githubusercontent.com/14074573/38772948-a97f4e6e-4018-11e8-85ea-bb0ee3d38be6.png)

Here is mine:
![meu](https://user-images.githubusercontent.com/14074573/38772954-cda450fa-4018-11e8-9bc5-f529d42d3627.png)

Maybe is a hardware lack of information in the datasheet, but I'm not understanding how to correspond pins:

```
//           Mine Original
#define RB 7 // RB 7 - I thin k it is BYTE# pin  
#define RE 8 // RE 8 -  Is RP# equivalent to RE?        
#define CE 9 // CE 9 - I think CE0 is similar to CE with CE1 and CE2 grounded 
#define CLE ? // CLE 16 - I have no idea  
#define ALE ? //ALE 17 - I have no idea (maybe A0?)
#define WE 18 // WE 18 - Same definition WE#
#define WP       19
```
I don't know if it is really compatible. I'm also having trouble to understand this page:
I will try to compare deeply both datasheets.

**Comment by jallBreak** [comment]
hi please import to Arduino Leonardo R3 thank you
